cordova-lib/src/plugman/registry/registry.js

```diff
@@ -364,6 +364,26 @@ function fetchPlugin(plugin, client, useNpmRegistry) {
     });
 }
 
+function processPluginVersion(plugin) {
```

Comment:
I'm intentionally not doing that... Calling getLatestMatchingNpmVersion is expensive - I only use it for platform because we were already calling it..